manifests/b/BlastApps/FluentSearch/0.9.87.1/BlastApps.FluentSearch.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: BlastApps.FluentSearch
+PackageVersion: 0.9.87.1
+PackageLocale: en-US
+Publisher: BlastApps

```

Comment:
Publisher is blank in ARP (Programs and Features) could you fix this? Its required for `winget list` and `winget upgrade` to work.